### DISCUSSION TITLE
How to obtain the visible screen size of the client browser

### DISCUSSION THREAD

**Discussion by wasabismile** [zauberzeug/nicegui#3087] [Q&A]
### Question

I want to get the height of the client's screen and set the value as the height of element.  But, It do not work.  
```python
print(test_h) # not return the value of innerHeight
``` 
```python
from nicegui import ui
async def get_client_height():
    h= await ui.run_javascript('window.innerHeight')
    return h
test_h=get_client_height()
print(test_h)
ui.column().classes('bg-gray-100 w-full').style(f'height:{test_h}px')
#ui.column().classes('bg-gray-100 w-full').style(f'height:100px')

ui.run()
```

**Comment by falkoschindler** [comment]
Hi @wasabismile,

Usually you would use the CSS unit "vh" to scale the height of an element relative to the viewport height:

```py
.style('height:100vh')
```

Or you can use the corresponding Tailwind class:
```py
.classes('h-screen')
```

Apart from that there are multiple reasons why your code won't work:
1. You're not awaiting `get_client_height()`.
2. You're not allowed to await JavaScript calls on the auto-index page. See #2951 and #2983 for more background about this limitation.
